bf2fc6cc711aee1a0c2a/kas-fleet-manager
# Add service max capacity endpoint

## Description

  Closes https://issues.redhat.com/browse/MGDSTRM-2251.
    
~~ This is based on the maximum capacity query capability added by https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/120 to close https://issues.redhat.com/browse/MGDSTRM-2250.~~
    
 ~~Let's merge this PR once the #120 has been merged, otherwise all reviews should focus on the second commit.~~


## Verification Steps


1. Make sure the added integration tests are passing. 
2. Run the swagger docs locally with `make run/docs` and make sure that the new endpoint is there

## Type of change


## Checklist (Definition of Done)
